app/components/chart-filters-list.tsx

```diff
@@ -18,9 +40,26 @@ export const ChartFiltersList = ({
   dataSource: DataSource;
   chartConfig: ChartConfig;
 }) => {
+  const classes = useStyles();
   const locale = useLocale();
   const timeFormatUnit = useTimeFormatUnit();
 
+  const { setOpen, setActiveSection, setSelectedDimension } =
+    useMetadataPanelStore(
+      (state) => ({
+        setOpen: state.setOpen,
+        setActiveSection: state.setActiveSection,
+        setSelectedDimension: state.setSelectedDimension,
+      }),
+      shallow
+    );
+
+  const handleClick = useEvent((d: DimensionMetadataFragment) => {
+    setOpen(true);
+    setActiveSection("data");
+    setSelectedDimension(d);
+  });
```

Comment:
I wonder if we should not have a method openDimension in the store that would take care of the internals of activeSection, open-ness, and selected dimension ? This way it's not the component job to know about this.